src/_imagingmorph.c

```diff
@@ -129,46 +124,39 @@ apply(PyObject *self, PyObject *args) {
 static PyObject *
 match(PyObject *self, PyObject *args) {
     const char *lut;
-    PyObject *py_lut;
-    Py_ssize_t lut_len, i0;
+    PyObject *i0;
+    Py_ssize_t lut_len;
```

Comment:
```suggestion
    Py_ssize_t lut_len;
    PyObject *i0;
```
It seems better to have `lut` and `lut_len` next to each other?